packages/@vuepress/markdown/index.js

```diff
@@ -20,7 +20,9 @@ const tocPlugin = require('markdown-it-table-of-contents')
 const {
   slugify: _slugify,
   parseHeaders,
-  logger, chalk, normalizeConfig,
```

Comment:
Could you revert the changes about formatted code?

Reply:
it is a format from the code it should happen i think

Reply:
I don't see any config for code format in the vuepress repos. In this case, I think we should not do changes about code formatting in PRs to avoid unnecessary changes.

Reply:
i will revert that change yeah